kotlin-native/litmus/src/commonMain/kotlin/AffinityBindings.kt

```diff
@@ -0,0 +1,17 @@
+import kotlin.native.concurrent.Worker
+
+interface AffinityManager {
+    fun setAffinity(w: Worker, cpus: Set<Int>)
+    fun getAffinity(w: Worker): Set<Int>
+
+    fun scheduleShort2(): List<AffinityMap>
+    fun scheduleLong2(): List<AffinityMap>
+}
+
+expect fun getAffinityManager(): AffinityManager?
+
+fun List<AffinityMap>?.orUnrestricted(actorCount: Int): List<AffinityMap> {
+    return this ?: affinityScheduleUnrestricted(actorCount)
+}
```

Comment:
Maybe it would make sense to put all Affinity related functions/classes into single file? Currently it is hard to make sense of what these functions do. Some minimal KDoc at least would help.